test-cases/OWNERS

```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- mikenairn
+- pmccarthy
+- maleck13
+- philbrookes
+- grdryn
+- david-martin
```

Comment:
Think this is probably duplication as any [root-level OWNERS](https://github.com/integr8ly/integreatly-operator/blob/master/OWNERS) will be able to approve PRs in the test-cases repo also

Reply:
Yes, I thought so, but I saw that they are also in tests/OWNERS so maybe if the owners are not inheriting.. I don't know.. should I remove them?

Reply:
@damienomurchu is correct. I am a root-level OWNER but not an OWNER of the test folder https://github.com/integr8ly/integreatly-operator/blob/master/test/OWNERS but I have rights to merge PRs there.

Reply:
@b1zzu yes, should be fine to remove